### PR TITLE
Allow using webinterface without internet

### PR DESCRIPTION
**Pull request by kjyv**
* load js libraries from filesystem instead of internet by default (change NOINTERNET in code for previous behaviour)
* use uPlot instead of plotly (48kb instead of ~950kb) so everything actually fits on the filesystem
* add clevercoffee logo and all necessary js, css and woff files

### CODE REVIEW

File: src/userConfig_sample.h [hunk 1]
```diff
@@ -49,6 +49,7 @@ enum MACHINE {
 #define PASS "CleverCoffee"        // default password for WiFiManager
 #define MAXWIFIRECONNECTS 5        // maximum number of reconnection attempts, use -1 to deactivate
 #define WIFICONNECTIONDELAY 10000  // delay between reconnects in ms
+#define NOINTERNET 1               // Use libraries for the webinterface from the internet (0) or from the local filesystem (1). 0 has slightly faster load times
```
Comment (murmeltier08) [inline src/userConfig_sample.h:52]: We have CONNECTMODE = 0 (offline). What is the different to NOINTERNET?

Comment (kjyv) [reply src/userConfig_sample.h:52]: The difference is that CONNECTMODE == 0 does not attempt to connect to Wifi while this is just not loading any dependencies for the website from the internet. Maybe we can name them differently to distinguish better

**Review by fiendie — APPROVED** [review]

File: src/userConfig_sample.h (hunk 1 above)
Comment (fiendie) [reply src/userConfig_sample.h:52]: I think we should remove this line and make it the default as we discussed.

Comment (kjyv) [reply src/userConfig_sample.h:52]: If we're sure that most people won't need the previous behaviour then sure. We can just allow changing it in the code for those rare cases

**Review by fiendie — DISMISSED** [review]

**Comment by murmeltier08** [comment]
I get following message. I do erase flash, load the new filesystem, try to upload the firmware:
Building in release mode
Retrieving maximum program size .pio/build/esp32_usb/firmware.elf
Checking size .pio/build/esp32_usb/firmware.elf
Advanced Memory Usage is available via "PlatformIO Home > Project Inspect"
Error: The program size (1258937 bytes) is greater than maximum allowed (1245184 bytes)
*** [checkprogsize] Explicit exit, status 1
RAM:   [=         ]  10.8% (used 57624 bytes from 532480 bytes)
Flash: [==========]  101.1% (used 1258937 bytes from 1245184 bytes)

**Comment by kjyv** [comment]
Do you still have something else in the filesystem folder? I only see about 850kb that are used.

**Comment by kjyv** [comment]
Please try again, should be ok also with influx now

**Comment by murmeltier08** [comment]
Works now fine.

**Pull request merged**